### PR TITLE
Add osu!mania 10K+ playstyles article

### PR DESCRIPTION
**Pull request by cl8n**
new prereq to #10138, because they wanted to have more info about these things somewhere to link to.

has been reviewed by all of the involved 10k+ people and a few more already

### CODE REVIEW

File: wiki/Beatmapping/osu!mania_10K_plus_playstyles/en.md [hunk 1]
```diff
@@ -0,0 +1,104 @@
...
+
+![](img/DP-keys.png)
+
+**DP** or **Double Play**, named after the play mode in *[beatmania IIDX](https://en.wikipedia.org/wiki/Beatmania_IIDX "\"beatmania IIDX\" on Wikipedia")*, is a 14-key or 16-key playstyle where each hand is responsible for 7 keys, arranged into lower rows of 4 keys and upper rows of 3 keys. The lower and upper rows are interleaved such that the first key is lower, the second is upper, the third is lower, and so on. An additional *scratch*<!-- TODO: link --> key may be included on each outer edge of the stage.
+
+Along with [EZ2AC](#EZ2AC), this playstyle is more broadly referred to as "14K" or "14K2S".
+
+![](img/DP-hands.png)
+
+Each hand usually rests on the upper 3, lower outermost, and lower innermost keys of its side of the stage. The lower outer-middle key can be hit with the ring finger, and the lower inner-middle key can be hit with the thumb or index finger. The two scratch keys, if included, are commonly hit by moving a pinky finger outward.
```
[the remark this answers is not in the document]
Comment (cl8n) [reply wiki/Beatmapping/osu!mania_10K_plus_playstyles/en.md:70]: i tried numbering them in keybind order at first but it gets weird with the optional scratch cuz all the middle numbers have to shift by 1

maybe doesn't matter to renumber them... i'll try it

[the remark this answers is not in the document]
Comment (Walavouchey) [reply wiki/Beatmapping/osu!mania_10K_plus_playstyles/en.md:70]: could maybe omit scratch columns from the numbering

[the remark this answers is not in the document]
Comment (cl8n) [reply wiki/Beatmapping/osu!mania_10K_plus_playstyles/en.md]: did a mirrored numbering thing instead. i think the benefit of left-to-right numbering *would have* been that it matches the kind of numbering you see in modding and skinning etc, but the optional scratches mess that up anyway

i also didnt know how to Torus, i used Inter Bold instead and it looks fine to me

File: wiki/Beatmapping/osu!mania_10K_plus_playstyles/en.md [hunk 2]
```diff
@@ -0,0 +1,104 @@
...
+
+![](img/DP-keys.png)
+
+**DP** or **Double Play**, named after the play mode in *[beatmania IIDX](https://en.wikipedia.org/wiki/Beatmania_IIDX "\"beatmania IIDX\" on Wikipedia")*, is a 14-key or 16-key playstyle where each hand is responsible for 7 keys, arranged into lower rows of 4 keys and upper rows of 3 keys. The lower and upper rows are interleaved such that the first key is lower, the second is upper, the third is lower, and so on. An additional *scratch*<!-- TODO: link --> key may be included on each outer edge of the stage.
+
+Along with [EZ2AC](#EZ2AC), this playstyle is more broadly referred to as "14K" or "14K2S".
+
+![](img/DP-hands.png)
+
+Each hand usually rests on the keys labeled 1-2-4-6-7 in the diagrams above. Key 5 can be hit with the ring finger, and key 3 can be hit with the thumb or index finger. The two scratch keys, if included, are commonly hit by moving a pinky finger outward.
```
Comment (Walavouchey) [inline wiki/Beatmapping/osu!mania_10K_plus_playstyles/en.md:70]:
```suggestion
Each hand usually rests on the keys labelled 1, 2, 4, 6, and 7 in the diagrams above. Key 5 can be hit with the ring finger, and key 3 can be hit with the thumb or index finger. The two scratch keys, if included, are commonly hit by moving a pinky finger outward.
```

**Review by Walavouchey — APPROVED** [review]

**Pull request merged**